.github/workflows/release 2.yml

```
@@ -0,0 +1,82 @@
+
+on:
+  push:
+    tags: ["v[0-9]+.*"]
+
+permissions:
+  contents: write
+  id-token: write
+
+jobs:
+  ci:
+    uses: ./.github/workflows/ci.yml
```

actionlint (1.7.12)

[error] error while parsing reusable workflow "./.github/workflows/ci.yml": "workflow_call" event trigger is not found in "on:" at line:4, column:3

(workflow-call)